Jenkinsfile.content

```diff
@@ -9,10 +9,13 @@ node('vetsgov-general-purpose') {
 
   refStatusState = getGithubCommitState("vets-website", ref)
   if ("${refStatusState}" != "SUCCESS") {
-    slackSend(message:"Web content refresh on ${params.env} aborted due to '${refStatusState}' status on vets-website commit ${ref}.",
+    message = "Web content refresh on ${params.env} aborted due to '${refStatusState}' status on vets-website commit ${ref}."
+    echo "${message}"
```

Comment:
I just tested this format of interpolating this variable and it works and is more readable.